### PR TITLE
task

### PR DESCRIPTION
**Pull request by MariiaSuper**
https://MariiaSuper.github.io/layout_catalog/
https://MariiaSuper.github.io/layout_catalog/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,312 @@
...
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
```
Comment (OleziO) [inline src/index.html:27]: Make links more meaningful

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,312 @@
...
+          </li>
+
+          <li class="nav__item">
+            <a
+                  href="#"
+                  class="nav__link"
+                  data-qa="nav-hover"
+                >
+                  laptops & computers
+            </a>
```
Comment (OleziO) [inline src/index.html:41]: Fix formatting:

```
<a
  href="#"
  class="nav__link"
  data-qa="nav-hover"
>
  laptops & computers
</a>
```

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,103 @@
+/* #region card */
```
Comment (OleziO) [inline src/styles/blocks/card.scss:1]: Remove all comments from the code.

**Review by OleziO — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (MariiaSuper) [reply src/index.html:27]: what do you mean?

File: src/styles/utils/_normalize.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
+body {
+  margin: 0;
+}
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", Arial, sans-serif;
```
Comment (slavanosatskyi) [inline src/styles/utils/_normalize.scss:6]: font-family should be defined in body as it is inherited property

File: src/styles/blocks/header.scss [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  }
+}
+
+.image {
+  margin: 0;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
```
Comment (slavanosatskyi) [inline src/styles/blocks/header.scss:20]: nav block and its elements should be defined in a separate file

File: src/styles/blocks/card.scss [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $card-width;
+  background: $back-color;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-right: 24px;
```
Comment (slavanosatskyi) [inline src/styles/blocks/card.scss:11]: do not use external geometry on a block level

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]
Screenshot 2023-05-09 at 17 42 47
Try to fix your grid for cases when there are less then 4 elements per line

File: src/styles/utils/_normalize.scss (hunk 4 above)
Comment (MariiaSuper) [reply src/styles/utils/_normalize.scss:6]: I ailing elements by center? Do not see design or requirements in the task - how to align them (left, center, right, ...) ? Thanks

File: src/index.html [hunk 7]
```diff
@@ -5,8 +5,312 @@
...
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo" >
+        <img
+              src="images/logo.png"
+              alt="logo"
+              class="image"
+            >
```
Comment (anstsot) [inline src/index.html:17]: fix the tabulation
```suggestion
        <img
          src="images/logo.png"
          alt="logo"
          class="image"
        >
```

File: src/index.html [hunk 8]
```diff
@@ -5,8 +5,312 @@
...
+          <div class="card__price">
+            <p class="card__price-price">Price:</p>
+            <p class="card__price-cost">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__buttom"
+            data-qa="card-hover">
+            Buy
+          </a>
```
Comment (anstsot) [inline src/index.html:93]:
```suggestion
          <a
            href="#"
            class="card__buttom"
            data-qa="card-hover"
          >
            Buy
          </a>
```

File: src/styles/blocks/star.scss [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background: url(../images/star-active.svg) no-repeat center;
+}
+
+.stars__star:last-child {
```
Comment (anstsot) [inline src/styles/blocks/star.scss:14]: please use the proper class names like `.stars--1`, `.stars--4` etc.

File: src/styles/blocks/star.scss [hunk 10]
```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.stars__star {
```
Comment (anstsot) [inline src/styles/blocks/star.scss:7]: use the power of nesting
```suggestion 

  $__star {
```

**Review by anstsot — CHANGES_REQUESTED** [review]
It’s almost done! Let’s improve a little
Please check your tests as well
If you have any questions, feel free to ask them in the chat

**Comment by MariiaSuper** [comment]
yesterday I asked for help in the chat (with tests) but they did not help me. I did what was written but no result.What shall I do? who can help me?

File: src/index.html (hunk 1 above)
Comment (volodymyr-soltys97) [reply src/index.html:27]: Example for link apple href must be `#apple`

File: src/index.html [hunk 11]
```diff
@@ -5,8 +5,297 @@
...
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-price">Price:</p>
+            <p class="card__price-cost">$2,199</p>
+          </div>
+          <a
+          href="#"
+          class="card__buttom">
+        Buy
```
Comment (volodymyr-soltys97) [inline src/index.html:178]: Remember about code style. Check it everywhere
```suggestion
          <a
            href="#"
            class="card__buttom"
          >
            Buy
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Goob job.
But if there are problems with the tests, write in the chat and we will try to help

**Pull request closed without merging**